.github/workflows/cron_nightly.yml

```diff
@@ -0,0 +1,61 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron:  '30 2 * * *'
+
+jobs:
+  nightly:
+    runs-on: ubuntu-latest
+    name: Nightly
+    strategy:
+      matrix:
+        php: [ '7.3' ]
+    env:
+      GC_INSTANCE_NAME: ${{ secrets.GC_INSTANCE_NAME }}
+      GC_PROJECT_ID: ${{ secrets.GC_PROJECT_ID }}
+      GC_SERVICE_KEY: ${{ secrets.GC_SERVICE_KEY }}
+      GC_ZONE: ${{ secrets.GC_ZONE }}
+      GH_BRANCH: 'develop'
```

Comment:
What about 1.7.7.x?

Reply:
Must be managed in the matrix :D

Reply:
No the 177X is always managed by Travis CI as we move GA only on develop.

Reply:
It can be done in this PR, nothing is executed on the repository, everything is on GCP :)

Reply:
I agree nothing blocks us to handle 177 here as well, and this way the workflow will already be able to handle two versions which will be a must have once the 1.7.8.x branch is created anyway

Reply:
@Progi1984 ping :)

Reply:
`1.7.7.x` continue to use Travis CI. So we don't modify that.

`develop` and `1.7.8.x` will use GA... and we can stop Travis CI.

Reply:
But as @jolelievre said, it would be great to make the workflow ready for multiple versions, using matrix I guess.

Reply:
It will be done when we release `1.7.8.x` branch.

Reply:
It's not in the scope of the PR.

Reply:
I don't really agree, why did you put php 7.3 in the matrix then? It should be the same for the branch :)

Reply:
@atomiix Because it's the PHP version defined in `.travis.yml` : https://github.com/PrestaShop/PrestaShop/blob/develop/.travis.yml#L44

Reply:
It's in the travis yaml because in travis you have to define a PHP version, but not mandatory anymore in GA

Reply:
But since we create a release on line 40, we have to define a PHP version.

Reply:
I don't think putting 7.3 in the php key of the matrix actually define the php version that will be used though

Reply:
Oh you're right

Reply:
Fixed